docs/src/modules/java/pages/kickstart.adoc

```diff
@@ -165,6 +165,55 @@ The Maven failsafe plugin runs the integration tests when the `it` profile is en
 mvn verify -Pit
 ----
 
+=== Unit Tests for Event Sourced Entities with autogenerated Testkit
```

Comment:
Let's have a separate page for the testkit. The kickstart is rather brief about everything so we don't have to explain it here.

Reply:
What about adding it to "Implementing Event Sourced Entities" page? as in https://github.com/lightbend/akkaserverless-java-sdk/pull/335

Reply:
that would be good

Reply:
great. Done now